test/JDBC/input/BABEL-4554-vu-prepare.sql

```diff
@@ -0,0 +1,7 @@
+-- Verify that newly created temp tables have toasts in ENR. 
+
+CREATE TABLE #babel_4554_temp_table(a varchar(4000), b nvarchar(MAX), c sysname)
```

Comment:
can we add more test with index? also test with temp table but not in ENR, so toast shouldnt be in enr either

Reply:
added!